test/tools/unified-spec-runner/operations.ts

```diff
@@ -245,7 +245,13 @@ operations.set('deleteOne', async ({ entities, operation }) => {
 operations.set('dropCollection', async ({ entities, operation }) => {
   const db = entities.getEntity('db', operation.object);
   const { collection, ...opts } = operation.arguments;
-  return await db.dropCollection(collection, opts);
+  try {
+    await db.dropCollection(collection, opts);
```

Comment:
1) I think this should still `return` the result, instead of just calling the operation, to keep consistent  with the previous behavior, in case there is something that is expecting a particular boolean response
2) I am not convinced that try/catch *here* is a good approach - this is a general definition for the drop operation, so swallowing the error would mean we can't test for cases where we definitely should *not* be getting this error from `dropCollection`: can you explain again why we need this and can we find a more surgical place to insert it? perhaps with a ticket to remove at a later point?